bool verbose
---
bool suc
string result
string message